resources/web/template.html

```diff
@@ -262,107 +257,14 @@
 
     <div class="main-container">
       <section id="content" >
-        <div class="content-wrapper">   
+        <div class="content-wrapper">
 
-          <section id="guide">
+          <section id="guide" <!-- DOCS LANG -->>
```

Comment:
This one is a little funny this is how the perl script knows where to insert things.